lib/massager_filename_generator.ex

```diff
@@ -0,0 +1,33 @@
+defmodule MassagerFilenameGenerator do
+  # For example, this will turn: "input_path/order_1.csv" into {"input_path/order_1.csv", "input_path/order_01.csv"}
```

Comment:
This is the start of making things generic. Really I just need to know these specific parameters and I can figure out the rest.